src/StrokeEngine.cpp

```diff
@@ -557,169 +183,43 @@ String StrokeEngine::getPatternName(int index) {
     
 }
 
-void StrokeEngine::setMaxSpeed(float maxSpeed){
-    // Update pattern with new speed limits
-    if (xSemaphoreTake(_patternMutex, portMAX_DELAY) == pdTRUE) {
-        // Convert speed into steps
-        _maxStepPerSecond = int(0.5 + _motor->maxSpeed * _motor->stepsPerMillimeter);
-        patternTable[_patternIndex]->setSpeedLimit(_maxStepPerSecond, _maxStepAcceleration, _motor->stepsPerMillimeter);
-        xSemaphoreGive(_patternMutex);
-    }
-}
-
-float StrokeEngine::getMaxSpeed() {
-    return float(_maxStepPerSecond / _motor->stepsPerMillimeter);
-}
-
-void StrokeEngine::setMaxAcceleration(float maxAcceleration) {
-
-    // Update pattern with new speed limits
-    if (xSemaphoreTake(_patternMutex, portMAX_DELAY) == pdTRUE) {
-        // Convert acceleration into steps
-        _maxStepAcceleration = int(0.5 + _motor->maxAcceleration * _motor->stepsPerMillimeter);
-        patternTable[_patternIndex]->setSpeedLimit(_maxStepPerSecond, _maxStepAcceleration, _motor->stepsPerMillimeter);
-        xSemaphoreGive(_patternMutex);
-    }    
-}
-
-float StrokeEngine::getMaxAcceleration() {
-    return float(_maxStepAcceleration / _motor->stepsPerMillimeter);
-}
-
 void StrokeEngine::registerTelemetryCallback(void(*callbackTelemetry)(float, float, bool)) {

```

Comment:
Just there for my debugging. I really like your idea with a "fake" motor which is graphically representing what is happening. Could be a great replacement.

Reply:
Yeah, once I have the code a bit more sorted out, I definitely want to see if I can easily build a dashboard for graphing of internal motor parameters. I would be able to build a dummy motor then, and show the internal motion it would be expected to take.